src/sentry/utils/email/message_builder.py

```diff
@@ -181,7 +187,12 @@ def build(self, to, reply_to=None, cc=None, bcc=None):
 
         return msg
 
-    def get_built_messages(self, to=None, cc=None, bcc=None):
+    def get_built_messages(
+        self,
+        to: list[str] | None = None,
+        cc: Iterable[str] | None = None,
```

Comment:
Why is `to` a list but `cc` and `bcc` are `Iterable`?

Reply:
Good question let me see if i can get it to match.